sig-TEMPLATE/README.md

```diff
@@ -0,0 +1,13 @@
+# SIG Template
+
+## Meetings and Channel
+
+Join our [slack channel]() in the Cilium Slack, or join our meetings on 
+* Wednesday, 08:00 PST - [Meeting Notes]()
+
+## Leadership
+
+## Leads
+
+The Leads are responsible to drive the operations and technical decisions of a SIG area.
```

Comment:
I think that this current wording is probably the most practical way to document this,  but when it comes to leads driving technical decisions, I'd expect them to uphold the [community values](https://github.com/cilium/community/blob/main/VALUES.md) particularly around consensus, quality and pragmatism. I think it's probably a healthy thing for the group to exercise the muscles of building consensus and not assuming that leads will make every decision on every effort in the SIG.